backend/auth/index.js

```diff
@@ -0,0 +1,57 @@
+const express = require("express");
+const bodyParser = require("body-parser");
+const FormData = require("form-data");
+const fetch = require("node-fetch");
+const { client_id, redirect_uri, client_secret } = require("./config");
+
+const config = require("./config");
+
+const app = express();
```

Comment:
We already get a backend/app.js file where the express instance is bootstrapped and listens on a port. We don't need another express instance here.
Please move your route into `backend/app.js`.

Reply:
Sorry bro, I pulled the main branch which was not the latest one, I've pushed the new request.